src/playground/Insights/ui/event.tsx

```diff
@@ -0,0 +1,49 @@
+import { cn } from "@/lib/utils"
+import { forwardRef } from "react"
+
+interface EventProps {
+  title: string
+  length: number
+  lengthUnit: string
+  from: string
+  until: string
```

Comment:
Don't you think we should leave this more open to `title` & `subtitle` props only? And then in Factorial we just form these texts based on the data that we have